src/languages/sqf.js

```diff
@@ -47,21 +77,23 @@ export default function(hljs) {
   };
 
   const KEYWORDS = [
-    'case', 'catch',
+    'break', 'breakWith', 'breakOut', 'breakTo',
```

Comment:
are breakWith, breakOut, breakTo, continue, continueWith really keywords?
They are really just script  commands.
But then again waitUntil and such are in here too... Guess its fine